website-custom-views/src/content/development/permissions.mdx

```diff
@@ -23,37 +23,21 @@ The `PERMISSIONS` variable contains a `View` and `Manage` properties:
 
 You can then use the `PERMISSIONS` variable to reference the permission in the application code.
 
-<Info>
-
-This feature is available from version `21.21.0` onwards.
+## With addtional permissions groups
```

Comment:
```suggestion
## Additional permissions groups
```

If the suggestion doesn't work, please fix the typo

Reply:
Thanks, Ryan.